emukit/bayesian_optimization/local_penalization_calculator.py

```diff
@@ -1,8 +1,10 @@
+from emukit.core.interfaces.models import IPriorHyperparameters
```

Comment:
```suggestion
from ..core.interfaces.models import IPriorHyperparameters
```

and move it a tad bit below, where the rest of relative imports are

Reply:
Done.